src/Data/ProgrammesDb/EntityRepository/ServiceRepository.php

```diff
@@ -53,4 +53,19 @@ public function findAllInNetworks(): array
 
         return $qb->getQuery()->getResult(Query::HYDRATE_ARRAY);
     }
+
+    public function findAllInNetwork(string $networkId): array
+    {
+        $qb = $this->createQueryBuilder('service')
+            ->select('service', 'network')
+            ->addSelect('CASE WHEN network.position IS NULL THEN 1 ELSE 0 AS HIDDEN hasPosition')
```

Comment:
I'm not entirely sure why this is needed for ordering. You are only selecting from one network. So how can you order by a value to do with the network?